pages/docs.css

```diff
@@ -237,253 +236,253 @@ body {
   flex-direction: column;
   gap: 0.5rem;
 }
-=======
-
-html, body {
-    font-family: Roboto, sans-serif;
-	overflow-anchor: none;
+======= html,
+body {
+  font-family: Roboto, sans-serif;
+  overflow-anchor: none;
 }
 
 .page-layout {
-    display: flex;
-    margin: auto;
-    overflow: clip;
-    overflow-x: hidden;
-    position: relative;
-    margin-top: 12%;
+  display: flex;
+  margin: auto;
+  overflow: clip;
+  overflow-x: hidden;
+  position: relative;
+  margin-top: 12%;
 }
 
 .drawer__sidebar--perm {
-    margin: 0;
-    margin-left: 0;
-    width: 20%;
-    height: 90vh;
-    position: fixed;
-    top: 20%;
-    box-shadow: var(--secondary-box-shadow);
-}
-    
+  margin: 0;
```

Comment:
No need to use margin left if you are using margin. Shorthand would be best here.